Projects/DbCLI/SearchDb.py

```diff
@@ -510,114 +515,118 @@ def poolFromCurs(curs, similarityMethod):
 
 # ---- ---- ---- ----  ---- ---- ---- ----  ---- ---- ---- ----  ---- ---- ---- ----
 import os
-from optparse import OptionParser
-parser = OptionParser(_usage, version='%prog ' + _version)
-parser.add_option(
+import argparse
+
+parser = argparse.ArgumentParser(usage)
+parser.add_argument('-v', '--version', action='version', version='0.14.0')
+
+parser.add_argument('filename')
```

Comment:
Same thing: optional or required? Add some help text too.

Reply:
`parser.add_argument('filename')` - makes it mandatory to add a filename.